src/clang.rs

```diff
@@ -393,7 +393,7 @@ impl Cursor {
     /// parameters.
     pub fn args(&self) -> Vec<Cursor> {
```

Comment:
Probably propagating the error up using `try!` would be better. That way we can more granularly control the error messages.